src/main/java/org/openlmis/integration/dhis2/web/server/ServerController.java

```diff
@@ -57,21 +59,54 @@
 public class ServerController extends BaseController {
```

Comment:
Is authorization (user needs a role to read/create/update/delete) missing or are they handled other way or it's left for later?

Reply:
Left for later in SELV3-577